CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+
+### Breaking Changes
+
+* None.
+
+### Enhancements
+
+* Add support for limiting the number of objects matched by a query-based sync
```

Comment:
What happens if you run a server with an older ROS?

Reply:
Query-based sync will throw an error saying LIMIT isn't supported.

Reply:
That should then be documented in the methods.

Reply:
I would rather add a compatibility section in the changelog than document stuff in the API docs.

Reply:
I think it makes sense to mention it in the API docs for these specific methods because we aren't bumping the minimum version of ROS supported; you just can't use these specific functions with older ones.